Vagrantfile

```diff
@@ -14,17 +14,35 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo apt-get update
-    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
+    sudo apt-get install -y autoconf automake build-essential expect git libfreetype6-dev libpng12-dev libtool nmap openjdk-7-jdk openjdk-7-jre pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
     sudo pip install ipython[notebook] pandas
     sudo apt-get remove -y w3m
 
     ln -s /vagrant /home/vagrant/lisa
+
+    cd /home/vagrant/lisa
+    if [ ! -e ./tools/android-sdk-linux ]; then
+        echo Downloading Android SDK...
+        wget -q https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
```

Comment:
Should we add wget to the list of apt-get installed packages... just to be sure...